<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1342</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QGraphicsView" name="graphicsView">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>600</height>
    </rect>
   </property>
  </widget>
  <widget class="QTableWidget" name="tablaCuerpos">
   <property name="geometry">
    <rect>
     <x>820</x>
     <y>10</y>
     <width>501</width>
     <height>231</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string>X</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Y</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Masa</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Radio</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Vx</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Vy</string>
    </property>
   </column>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>820</x>
     <y>509</y>
     <width>521</width>
     <height>81</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="Layout">
    <item>
     <widget class="QPushButton" name="btnAgregarCuerpo">
      <property name="text">
       <string>Agregar Cuerpo</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnSistema1">
      <property name="text">
       <string>Sistema 1</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnSistema2">
      <property name="text">
       <string>Sistema 2</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnCrearSistema">
      <property name="text">
       <string>Crear sistema</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnLimpiarTabla">
      <property name="text">
       <string>Limpiar sistema</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="formLayoutWidget">
   <property name="geometry">
    <rect>
     <x>820</x>
     <y>260</y>
     <width>108</width>
     <height>211</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>X :</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="inputX">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Y :</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="inputY">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Masa :</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="inputMasa">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>Radio :</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="inputRadio">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>Vx</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QLineEdit" name="inputVx">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>Vy :</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QLineEdit" name="inputVy">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
